userspace/libscap/engine/kmod/scap_kmod.c

```diff
@@ -253,11 +290,29 @@ int32_t scap_kmod_init(scap_t *handle, scap_open_args *oargs)
 		// (for subsequent devices it's a no-op thanks to the check above)
 		handle->m_schema_version = schema_version;
 
+
+	   /* Each data page is mapped twice to allow "virtual"
```

Comment:
🙃 we had that discussion in the other PR that got accidentally closed by the bot. Would say it's ok and these comments add clarity, but any place is ok to add it would say, just so it's clear that user defined param is double-mapped.

Reply:
Perhaps the option could be translated with a `scap_engine_util` function in `scap_open_live_int`?
This way, we only have a single point where we translate the option to its real value.

Reply:
I think I will move it directly in sinsp when we perform the validation in a unique point, it seems a great place to set it since we already check the number of pages, WDYT?

Reply:
I've moved the validation in every single engine, so into `check_per_cpu_buffer_num_pages`